test/test_ufbrw.F90

```diff
@@ -104,7 +107,9 @@ program test_ufbrw
 
       do while(ireadmg(20,subset,idate)==0)
         do while(ireadsb(20)==0)
+          if(i==1) call openbf(20,'QUIET',2)
```

Comment:
Shouldn't the 3rd argument now be 3 instead of 2?  Otherwise, how would the new `if(iprt>=3) then` lines in invwin() ever get triggered?

Reply:
My motive here is to use print level 2 to write information from ufbrw. The lower level print from for example invwin would all be available at print level 3 if required. .

Reply:
OK, but if we're going to add these lines of code to the library, then we should have a corresponding test somewhere.  Otherwise how do you know that the new lines of code you've added to invwin() are really working the way you intend?